.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: CI for packaging
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: install dependencies
+        run: |
+          sudo apt-get install -y rpm make ruby ruby-dev rubygems build-essential
+          sudo gem install --no-document fpm
+          sudo apt-get install -y rpm
+
+      - name: run test
+        run: |
+          make package type=rpm app=apisix version=2.3 code_tag=2.3
```

Comment:
why use ubuntu for rpm package？

Reply:
Github action only supports ubuntu when we choose to use Linux. If we need centos environment, we need to do that in docker.

Reply:
Fpm can be packaged across platforms. I find it more convenient to install fpm with ubuntu, which is easier for testing.